ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDialogOptions.kt

```diff
@@ -82,6 +86,10 @@ data class PaywallDialogOptions internal constructor(
             this.listener = listener
         }
 
+        fun setMyAppPurchaseLogic(myAppPurchaseLogic: MyAppPurchaseLogic?) = apply {
```

Comment:
I noticed we don't have docs for a few of these... However, it would be good to add some docstring here explaining what this does (same for the `PaywallOptions` one, and that it will be ignored unless SDK is using `PurchasesAreCompletedBy.MyApp`